mockmaker/bytebuddy/main/java/org/mockito/internal/creation/bytebuddy/MockBytecodeGenerator.java

```diff
@@ -61,18 +49,16 @@ public MockBytecodeGenerator() {
                              .intercept(to(MockMethodInterceptor.ForWriteReplace.class));
         }
         return builder.make()
-                      .load(SearchingClassLoader.combineLoadersOf(allMockedTypes(features)), ClassLoadingStrategy.Default.INJECTION)
+                      .load(new MultipleParentClassLoader.Builder()
+                              .append(features.mockedType)
+                              .append(features.interfaces)
+                              .append(MultipleParentClassLoader.class.getClassLoader())
+                              .append(Thread.currentThread().getContextClassLoader())
+                              .filter(isBootstrapClassLoader())
+                              .build(), ClassLoadingStrategy.Default.INJECTION)
```

Comment:
The class search path of this class loader ia now fully identical to that of the searching one. It just aggregates all parents within a single class loader instead of creating one per parent.